plenum/common/txn_util.py

```diff
@@ -225,14 +226,17 @@ def reqToTxn(req):
     if isinstance(req, str):
         req = json.loads(req)
     if isinstance(req, dict):
-        req = Request(
+        kwargs = dict(
             identifier=req.get(f.IDENTIFIER.nm, None),
             reqId=req.get(f.REQ_ID.nm, None),
             operation=req.get(OPERATION, None),
             signature=req.get(f.SIG.nm, None),
             signatures=req.get(f.SIGS.nm, None),
             protocolVersion=req.get(f.PROTOCOL_VERSION.nm, None)
         )
+        if req.get(f.FEES.nm):
```

Comment:
Plenum should not be aware of FEEs

Reply:
It does not know how to process fees or parse fees (structure) or anything else, it just knows that request and response can contain a section called `fees`. Its similar to indy-sdk having a notion of payment but exact details are left to the plugin